sc/social/like/tests/test_subscribers.py

```diff
@@ -220,6 +222,57 @@ def test_validate_with_package_uninstalled(self):
         # should not raise exceptions on editing
         notify(EditFinishedEvent(self.news_item))
 
+    @requests_mock.mock()
+    def test_validate_facebook_prefetch_valid(self, m):
+        RESPONSE_VALID = """{"share":{"comment_count":0,"share_count":4},
+                              "og_object":{"id":"442171799221474","description":"Plone","title":"Lorem ipsum","type":"website",
+                              "updated_time":"2013-10-31T12:59:59+0000"},"id":"http:\/\/nohost/plone/\/news-item"}"""
+        url = self.news_item.absolute_url()
+        m.post('https://graph.facebook.com/?id=' + url + '&scrape=true', text=RESPONSE_VALID, status_code='200')
+        api.portal.set_registry_record('facebook_prefetch_enable', True, interface=ISocialLikeSettings)
+
+        # testing log
+        expected = ('sc.social.like', 'INFO', u'Prefetching: http://nohost/plone/lorem-ipsum')
+        log = LogCapture(PROJECTNAME)
+
+        with api.env.adopt_roles(['Manager']):
+            api.content.transition(self.news_item, 'publish')
+
+        log.check(expected)
+
+    @requests_mock.mock()
+    def test_validate_facebook_prefetch_invalid(self, m):
+        RESPONSE_INVALID = """{"share":{"comment_count":0,"share_count":4},"id":"http:\/\/nohost/plone/\/news-item"}"""
```

Comment:
I'm not sure about this test; did you really found this happening? on which circumstances?

Reply:
In sites that Facebook does not have access to content, example localhost or intranet.